packages/react-composites/src/composites/CallWithChatComposite/components/AddPeopleDropdown.styles.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IContextualMenuStyles, Theme } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const themedCopyLinkButtonStyles = (mobileView: boolean | undefined, theme: Theme): Partial<IButtonStyles> => ({
+  root: {
+    minHeight: mobileView ? '3rem' : '2.5rem',
+    borderRadius: mobileView ? theme.effects.roundedCorner6 : theme.effects.roundedCorner4,
+    width: '100%'
+  },
+  textContainer: {
+    display: 'contents'
+  }
+});
+
+/**
+ * @private
+ */
+export const themedMenuStyle = (theme: Theme): Partial<IContextualMenuStyles> => ({
+  root: {
+    borderRadius: theme.effects.roundedCorner6
+  }
+});
+
+/**
+ * @private
+ */
+export const iconStyles = {
+  lineHeight: '0',
```

Comment:
Might be good to add a comment why thi `lineHeight` needed to be set to 0 (if we know).